pyp2rpm/metadata_extractors.py

```diff
@@ -28,6 +28,22 @@
 
 logger = logging.getLogger(__name__)
 
+def cut_to_length(text, length, delims):
+    """Shorten given text on first delimiter after given number
+    of characters.
+    """
+    if not isinstance(delims, list):
+        delims = [delims]
+
+    for delim in delims:
+        cut = text.find(delim, length)
+        if cut != -1:
+            break
```

Comment:
Is the issue that docstring doesn't match the implementation or the implementation itself? Only one delimiter is always used as an argument at this moment.